docs/_basic/ja_listening_actions.md

```diff
@@ -41,7 +41,7 @@ app.action({ action_id: 'select_user', block_id: 'assign_ticket' },
       const result = await app.client.reactions.add({
         token: context.botToken,
         name: 'white_check_mark',
-        timestamp: action.ts,
+        timestamp: action.action_ts,
```

Comment:
Good catch but this should be `body.message.ts` instead. `action_ts` may not be the same with message's ts.

Reply:
Oops, thanks for catching that